www/src/components/layout/WithNotifications.js

```diff
@@ -14,90 +12,50 @@ import Markdown from '../incidents/Markdown'
 
 import { NOTIFICATIONS_QUERY } from './queries'
 
-function WithNotifications({ children }) {
+export function WithNotifications({ children }) {
+  const [notifications] = usePaginatedQuery(NOTIFICATIONS_QUERY,
+    { variables: {} },
+    data => data.notifications)
+
+  return children({
+    notificationsCount: notifications.length,
+  })
+}
+
+export function NotificationsPanel({ closePanel }) {
   const [notifications, loadingNotifications, hasMoreNotifications, fetchMoreNotifications] = usePaginatedQuery(NOTIFICATIONS_QUERY,
     { variables: {} },
     data => data.notifications)
-  const [isPanelOpen, setIsPanelOpen] = useState(false)
+
+  if (!notifications.length) {
+    return (
+      <P>
+        You do not have any notifications yet.
+      </P>
+    )
+  }
```

Comment:
This needs some padding
<img width="546" alt="Screen Shot 2022-07-28 at 10 59 35 AM" src="https://user-images.githubusercontent.com/85062/181612921-e0421cd6-5b6b-471b-9c18-142503294b83.png">
.